### DISCUSSION TITLE
Silent install option?

### DISCUSSION THREAD

**Discussion by brandonaut** [jamulussoftware/jamulus#1487] [Technical Support & Help] [answered]
Is there an option for silent install when running the Jamulus Windows installer? Something like:

```
jamulus_3.7.0_win.exe /s
```

This would enable adding Jamulus as a package in WinGet (https://github.com/microsoft/winget-pkgs/discussions/3388)

**Comment by ann0see** [comment, marked as answer]
Yes, I think NSIS supports /S. Didn't try it for the installer, but the uninstaller supports it. 

Thanks for working on this!

Edit: @brandonaut just tested it via power shell and the installer also works with /S
